main.go

```diff
@@ -0,0 +1,193 @@
+// dataone-indexer
+//
+// This service listens to the AMQP exchange for the CyVerse data store for events that are relevant to the DataONE
+// member node service and records them in the DataONE event database.
+package main
+
+import (
+	"database/sql"
+	"strings"
+
+	"github.com/cyverse-de/configurate"
+	"github.com/cyverse-de/dataone-indexer/database"
+	"github.com/cyverse-de/dataone-indexer/logger"
+	"github.com/cyverse-de/dataone-indexer/model"
+	"github.com/cyverse-de/dbutil"
+	_ "github.com/lib/pq"
+	"github.com/spf13/viper"
+	"github.com/streadway/amqp"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+var defaultConfig = `
+amqp:
+  uri: amqp://guest:guest@rabbit:5672/jobs
+  exchange:
+    name: de
+  routing-key:
+    subscription: data-object.*
```

Comment:
Do you mean this to be `*` (matches one word), no `.`s, or `#` (matches anything)?

Reply:
(now that I've looked further this might be an obsolete comment, but it'll still apply if you keep the separate config for subscription vs. what messages to process)